pyper/task/__init__.py

```diff
@@ -1,8 +1,8 @@
-import abc
 from typing import Any, Callable
 
 from pydantic.main import BaseModel
 
 
 class Task(BaseModel):
     fn: Callable[..., Any]
+    fallback: Callable[[Exception], None] = lambda: None
```

Comment:
```suggestion
    fallback: Optional[Callable[[Exception], None]] = Field(default=None)
```
I think making fallback field optional is better for the sake of simplicity